src/v3/test/integration/util/createAuthClient.ts

```diff
@@ -155,5 +186,9 @@ export const createAuthClient = (options: CreateAuthClientOptions): OktaAuth =>
     'X-Okta-User-Agent-Extended': 'okta-auth-js/9.9.9',
   });
 
-  return authClient;
+  return {
+    authClient,
+    pauseMocks,
+    resumeMocks,
```

Comment:
I think these are going to be useful in integration tests to make sure network calls don't cause flakiness.